agent/agent.go

```diff
@@ -370,6 +373,8 @@ func (a *Agent) Run(ctx context.Context) error {
 		a.StartSocketListeners(ctx)
 	}
 
+	// REVIEW(ccressent): I really do not understand the comment below.  Added
+	// by Simon. I suggest deleting altogether?
```

Comment:
I believe those counters were removed in a previous bugfix or refactoring. I think it's safe to remove the comment.